src/main/resources/application.yaml

```diff
@@ -199,6 +199,10 @@ darts:
 
       response-cleanup-buffer-days: 0
 
+      batch-size: ${ARM_BATCH_SIZE:10}
```

Comment:
Dont think we need env variables. We can hardcode the properties here.

Reply:
I've left the env var as agreed in a separate conversation. It has been agreed that we'll have a follow up ticket to complete the setup of the env vars everywhere else is needed, since this was out of scope for this ticket